libraries/AP_Compass/AP_Compass_LSM303D.cpp

```diff
@@ -236,6 +236,11 @@ bool AP_Compass_LSM303D::_read_sample()
         return false;
     }
 
+    /* check for overrun */
+    if ((rx.status & 0x70) != 0) {
```

Comment:
No macros, huh. Then I looked at the whole file and there are several places like this. Ok, I think we can let that to another PR.